docs/readme.md

```diff
@@ -0,0 +1,13 @@
+# API documentation
+
+- [`journeys(from, to, [opt])`](journeys.md) – get journeys between locations
```

Comment:
this should be `journeys(origin, destination, [opt])` according to FPTF

Reply:
This PR just moved it from `/readme.md` into `/docs/readme.md`. Will do the change on `master`.

Reply:
See `docs/journeys.md` as well.